libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -124,16 +124,12 @@ class NavigationView @JvmOverloads constructor(
 
         MapboxNavigationApp.attach(this)
 
+        val componentActivity = context.toComponentActivity()
         attachCreated(
             AnalyticsComponent(),
-            LocationPermissionComponent(context.toComponentActivityRef(), navigationContext.store),
+            LocationPermissionComponent(context.toComponentActivity(), navigationContext.store),
             TripSessionComponent(lifecycle, navigationContext.store),
             MapLayoutCoordinator(navigationContext, binding),
-            OnKeyListenerComponent(
```

Comment:
Will you make it a `UIComponent`? That way it can be moved around with the rest of these components

Reply:
```
OnBackPressedComponent(componentActivity, navigationContext.store)
```

Reply:
Agreed 👍🏻